GW2EIEvtcParser/EncounterLogic/EncounterIDs.cs

```diff
@@ -36,6 +37,8 @@ public static class FractalMasks
         public const long SunquaPeakMask = 0x000300;
         public const long SilentSurfMask = 0x000400;
         public const long LonelyTowerMask = 0x000500;
+        public const long CaptainMaiTrinBossMask = 0x000700;
+        public const long DeepstoneMask = 0x000700;
```

Comment:
Uses the same mask as CaptainMaiTrin

Reply:
Also I'm not sure we want masks for those, we have no ambition to add official support for them.